### DISCUSSION TITLE
Can I Use mitmproxy to Stream Hotstar/SonyLIV?

### DISCUSSION THREAD

**Discussion by kapilsingh421** [mitmproxy/mitmproxy#6896] [Q&A]
I have WireGuard running, but I encounter an error when trying to access [www.hotstar.com](http://www.hotstar.com/). I'm considering using mitmproxy to troubleshoot and potentially resolve this issue. Can I use mitmproxy for this purpose?

Both are using "server: AkamaiGHost"

<img width="773" alt="hotstar" src="https://github.com/mitmproxy/mitmproxy/assets/37150928/00405f1d-e236-4c14-9455-5a2dd13b00e1">
<img width="899" alt="sonylive" src="https://github.com/mitmproxy/mitmproxy/assets/37150928/b36ed4ae-baa7-49c0-8076-a507963bc07a">

**Comment by Prinzhorn** [comment]
Potentially their WAF, see https://github.com/mitmproxy/mitmproxy/issues/4575
